.rubocop.yml

```diff
@@ -29,6 +29,9 @@ Style/TrailingCommaInArrayLiteral:
 Style/FrozenStringLiteralComment:
   Enabled: false
 
+Style/AsciiComments:
+  Enabled: false
```

Comment:
I'm happy with this but I'm curious about the rationale?

Reply:
Generally, I'd like to have a rationale for every disabled rule or any other variation from [the default](https://github.com/rubocop/rubocop/blob/3482522e2be022eee2321722f4e049ab4bcb3bf1/config/default.yml#L2721-L2728).

Reply:
My rationale is:
- Were already use some Unicode characters like em dash (`—`) — which I'm a big fan of, I have to admit — and ellipsis… which I use here and there too.
- It feels reasonable to consider that we might also want to use emojis, as well as some Unicode characters necessary to draw nice Monodraw graphs, in comments, and in the era of Unicode I don't see any reason why we shouldn't allow them.

I'll add a comment to explain that rationale in the config file indeed.

!["Jealous Girlfriend" meme variation with em dash](https://pbs.twimg.com/media/DqXQr62XcAAvpiU.jpg)

Reply:
Rationale added as comment in 70fb5d5a168126b3438351fc7b2328c5e20df4aa

Reply:
🤣 👍